trade-finance-manager-ui/component-tests/utilisation-reports/_macros/premium-payments-table.component-test.ts

```diff
@@ -73,65 +73,89 @@ describe(component, () => {
 
   it('should render all table headings', () => {
     const wrapper = getWrapper();
-    wrapper.expectElement(`${tableSelector} thead th`).toHaveCount(8);
-    wrapper.expectElement(`${tableSelector} thead th:contains("Facility ID")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Exporter")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Reported fees")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Reported payments")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Total reported payments")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Payments received")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Total payments received")`).toExist();
-    wrapper.expectElement(`${tableSelector} thead th:contains("Status")`).toExist();
+    wrapper.expectElement(`${tableSelector} thead th`).toHaveCount(9);
+    wrapper.expectText('[data-cy="premium-payments-table--facility-id-header"]').toRead('Facility ID');
+    wrapper.expectText('[data-cy="premium-payments-table--exporter-header"]').toRead('Exporter');
+    wrapper.expectText('[data-cy="premium-payments-table--reported-fees-header"]').toRead('Reported fees');
+    wrapper.expectText('[data-cy="premium-payments-table--reported-payments-header"]').toRead('Reported payments');
+    wrapper.expectText('[data-cy="premium-payments-table--total-reported-payments-header"]').toRead('Total reported payments');
+    wrapper.expectText('[data-cy="premium-payments-table--payments-received-header"]').toRead('Payments received');
+    wrapper.expectText('[data-cy="premium-payments-table--total-payments-received-header"]').toRead('Total payments received');
+    wrapper.expectText('[data-cy="premium-payments-table--status-header"]').toRead('Status');
+    wrapper.expectText('[data-cy="premium-payments-table--hidden-header"]').toRead('Facility ID and Exporter and Reported fees and Reported Payments');
+  });
+
+  it.each`
+    columnName             | selector
+    ${'Facility ID'}       | ${'[data-cy="premium-payments-table--facility-id-header"]'}
+    ${'Exporter'}          | ${'[data-cy="premium-payments-table--exporter-header"]'}
+    ${'Reported Fees'}     | ${'[data-cy="premium-payments-table--reported-fees-header"]'}
+    ${'Reported payments'} | ${'[data-cy="premium-payments-table--reported-payments-header"]'}
+  `("should set the '$columnName' column header to be aria-hidden", ({ selector }: { selector: string }) => {
+    const wrapper = getWrapper();
+
+    wrapper.expectElement(selector).toHaveAttribute('aria-hidden', 'true');
   });
 
-  it("should use the 'govuk-table__header--numeric' class for numeric columns", () => {
+  it('should render the alternative hidden header as visually hidden', () => {
```

Comment:
```suggestion
  it('should render the visually hidden alternative header', () => {
```

Reply:
The test above checks it is rendered just not that it is visually hidden, which is what this test is for, so I don't think that suggested name is accurate. But I will make the name more explicit.